source/query-with-natural-language/query-with-natural-language.txt

```diff
@@ -11,7 +11,54 @@ Query with Natural Language
    :backlinks: none
    :depth: 2
 
-Query with Natural Language
+You can use MongoDB Compass to generate filter queries and aggregations 
+using natural language. Compass uses AI to generate filter queries based 
+on prompts you provide. Querying with natural language can be a helpful 
+starting point and assist you in learning to write MongoDB queries.
+
+Use Cases
+---------
+
+You may want to use Compass natural language querying to:
+
+- Ask plain text questions about your data.
+- Create an initial query or aggregation pipeline that you can 
+  modify to suit your requirements.
+- Learn how to write complex queries with multiple aggregation stages.
+
+Behavior
+--------
+
+- Compass natural language querying is available starting in version 
+  ``1.40.x``.
+
+- Natural language querying utilizes `Azure Open AI 
+  <https://azure.microsoft.com/en-us/products/ai-services/openai-service>`__ 
+  as its current provider. Please note that this provider may be subject
```

Comment:
(nit)
We don't generally say please in technical writing ... You might just state: This provide is subject to change in future releases.